### PR TITLE
Avoid const in strict mode on old node

### PR DESCRIPTION
**Pull request by phated**
@jonschlinkert this shows the problems in old node.  We weren't seeing them due to `const` usage in strict mode.  I'll fix the problems inline with this PR.

### CODE REVIEW

**Comment by jonschlinkert** [comment]
Got it, thx!

**Comment by phated** [comment]
@jonschlinkert @doowb I've pushed fixes for the broken tests.  Please review and let me know what you think.  I'm guessing travis will take a long time due to OSX

**Comment by phated** [comment]
@jonschlinkert @doowb I've just pushed up symbol support for node 0.12 (which has symbols but not Object.assign).  I believe that should be the last problem.

File: index.js [hunk 1]
```diff
@@ -27,6 +27,16 @@ function assign(a, b) {
       a[key] = b[key];
     }
   }
+
+  if (typeof Object.getOwnPropertySymbols === 'function') {
```
Comment (jonschlinkert) [inline index.js]: you could use https://github.com/jonschlinkert/assign-symbols if you want. I don't mind either way, just wanted to mention it.

**Pull request merged**